KTLOSNaxxLoot.json

```diff
@@ -519,7 +519,7 @@
 			"wowID":"23070"
 		},
 		"Spire of Twilight":{
-			"prio":["Navia(1)","EP/GP"],
+			"prio":["Navia(1)","Dr(2)","EP/GP"],
```

Comment:
Not sure Thrane's druid has performed to the point where he should have prio on the 3rd best healing weapon.
Also he has 54 GP currently which means he's been winning a bunch of items, this should probably stay EPGP in r2 for now.